src/google-logger.mjs

```diff
@@ -24,7 +30,7 @@ export class GoogleLogger {
   }
 
   logRUM(json, id, weight, referer, generation, checkpoint, target, source, timePadding) {
-    if (!isValidCheckpoint(checkpoint) && !isReasonableWeight(weight)) {
+    if (!isValidCheckpoint(checkpoint) && !isReasonableWeight(weight) && !isValidId(id)) {
```

Comment:
@trieloff I feel like we should use OR instead of AND here

```js
if (!isValidCheckpoint(checkpoint) || !isReasonableWeight(weight) || !isValidId(id)) {
  return;
}
```

Reply:
Yes, I have another PR open for that

Reply:
This caused some tests to fail

Reply:
I will wait until that PR is merged